tests/benchmarks/latency_measure/Kconfig

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+mainmenu "Latency Measure Benchmark"
+
+source "Kconfig.zephyr"
+
+config BENCHMARK_NUM_ITERATIONS
+	int "Number of iterations to gather data"
+	default 10000
```

Comment:
maybe make this 1000 as the default to avoid running forever on some platforms in CI and use 10000 where you are measuring for real on real HW....

Reply:
Sounds reasonable.

Done.